backend/user_management/minio/routers/upload.py

```
@@ -0,0 +1,41 @@
+    if not product:
+        raise HTTPException(status_code=404, detail="Product not found")
+
+    urls = [await upload_to_minio(file, "photos") for file in files]
+    product.product_pic = urls  # Store URLs in PostgreSQL
+    db.commit()
+    return {"message": "Photos uploaded successfully", "urls": urls}
+
+@router.post("/upload/videos/{product_id}", response_model=ProductVideoUploadResponse)
+async def upload_videos(
+    product_id: int, 
+    files: List[UploadFile] = File(...), 
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)